helm-chart/splunk-operator/values.yaml

```diff
@@ -0,0 +1,137 @@
+  # Set resource requests and limits for kube-rbac-proxy container
+  resources:
+    limits:
+      cpu: 200m
+      memory: 100Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+
+# Splunk Operator configurations
+splunkOperator: 
+
+  # Enable to deploy Splunk Operator
+  enabled: true
+
+  # Splunk operator image and pull policy
+  # reference: https://github.com/splunk/splunk-operator
+  image:
+    repository: docker.io/splunk/splunk-operator:latest
+    pullPolicy: IfNotPresent
+
+  # Set image pull secrets to pull image from a private registry
+  imagePullSecrets: []
+
+  # Give operator cluster-wide access, otherwise access is limited to operator namespace
+  clusterWideAccess: true
+
+  # List specific namespaces to watch e.x "namespace1,namespace2", must have cluster-wide access
+  watchNamespaces: ""
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  # Add annotations to Splunk Operator deployment
+  annotations: {}
+
+  # Add pod annotations to Splunk Operator deployment pod
+  podAnnotations: {}
+
+  # Set security context for manager container, default ensures that no child process of manager can gain more privileges than manager
+  # reference: https://kubernetes.io/docs/concepts/security/pod-security-policy/#privilege-escalation
+  containerSecurityContext:
+    allowPrivilegeEscalation: false
+
+  # Set security context for Splunk Operator pod 
+  # reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#podsecuritycontext-v1-core
+  securityContext:
+    runAsUser: 1001
+    fsGroup: 1001
+    runAsNonRoot: true
+
+  # Set usage of host networking and namespace
+  # reference: https://kubernetes.io/docs/concepts/security/pod-security-policy/#host-namespaces
+  hostNetwork: false
+  hostPID: false
+  hostIPC: false
+
+  # Define liveness probe to check if manager container is running
+  # reference: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 8081
+    initialDelaySeconds: 15
+    periodSeconds: 20
+
+  # Define readiness probe to check if manager container is ready for requests
+  # reference: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 8081
+    initialDelaySeconds: 5
+    periodSeconds: 10
+
+  # Set manager service http port and define additional ports if necessary
+  # reference: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
+  service:
+    type: ClusterIP
+    httpPort: 80
+    additionalPorts: []
+  # - name: metric
+  #   port: 8080
+  #   protocol: TCP
+  #   targetPort: 8080
+  
+  # Set resource requests and limits for manager container
+  resources:
```

Comment:
can you take the values from existing splunk-operator (resource limits from manager.yaml)

Reply:
Added the existing resource values from the main branch.